app/scripts/components/app/add-raw.js

```diff
@@ -48,13 +49,13 @@ const AddRaw = React.createClass({
     }
   },
 
-  componentWillMount: function () {
+  UNSAFE_componentWillMount: function () {
```

Comment:
If we're going to go through the pain of the upgrade, I think we might as well move away from the deprecated methods. 

It seems like for this case, it would be recommended to use `getInitialState`. See https://reactjs.org/docs/react-without-es6.html#setting-the-initial-state

For other cases where we are just dispatching an action when the component is loading/loaded, it seems like `componentDidMount` is recommended. I don't love that, because `componentDidMount` fires slightly later than `componentWillMount` in the lifecycle, but perhaps it won't make a huge difference in practice

Reply:
On further review, the upgrade paths to the new lifecycle methods are not necessarily drop-in replacements, so perhaps for the sake of keeping this PR sane we should leave it as is and file a separate ticket for the re-factor to use the new lifecycle methods. Thoughts?

Reply:
Yes, agree that we should definitely take care of this. They did say this was going away completely after React v.17.
But for the sake of this PR not getting larger let's make that another ticket.

Reply:
Cool. Would you mind writing up that ticket?